scripts/release-bit-verification.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/sh
+
+############################################################################
+#  PREREQUISITES FOR THIS SCRIPT
+# 1. Redistributable-binary(.rpm) should be passed as the first argument
+# 2. Login to the cluster should be done prior to running this script
+# 3. The cluster should be in a state where it can be used for testing
+#
+# USAGE:
+# ./check-binary.sh redistributable-binary
+#
+# Example: ./check-binary.sh ~/Downloads/odo-redistributable-2.4.3-1.el8.x86_64.rpm
+#
+
+shout() {
+  echo "--------------------------------------------------------------------------"
+}
+# Check SHASUM for all the binary files and there should be no difference
+
+# Create a Temp directory 
+mkdir -p /tmp/temp
```

Comment:
I was thinking about using a CLI like  `mktemp`:

```
WORKING_DIR=$(mktemp -d)
```

Reply:
Ooh  yes! That makes more sense. Updating it ASAP

Reply:
Done!